ansible/roles/proxy/tasks/main.yml

```diff
@@ -25,3 +25,10 @@
   file: path={{ tarballs_dir }} state=directory
 - name: "copy /etc/hosts to proxy"
   template: src=roles/proxy/templates/etc_hosts dest=/etc/etc_hosts owner=root group=root mode=0644
+# Update ca-certificates to ensure that CA certificates are new enough
+# to support SSL verification against websites used to download software
+- name: "update ca-certificates package to latest version"
+  yum:
+    name: 'ca-certificates*'
+    state: latest
+    update_only: yes
```

Comment:
update-only is a bit redundant here, since it will always be installed. I'm pretty sure it's part of the core packages.

Reply:
I figured it was better to be safe since I added the wildcard. Would you prefer I remove it? If so, should I remove the wildcard too and limit the package to ca-certificates only?

Reply:
OBE if you update to latest CentOS 7 in the example props.

I think the wildcard should be dropped, because any other matching RPMs would be unexpected and who knows what they might contain. Keeping `update_only` is safer with the wildcard. Dropping the wildcard, I don't have a strong opinion about `update_only`. It'd be redundant, but harmless. In any case, all this is OBE if you update to the latest CentOS 7, which I think is better.